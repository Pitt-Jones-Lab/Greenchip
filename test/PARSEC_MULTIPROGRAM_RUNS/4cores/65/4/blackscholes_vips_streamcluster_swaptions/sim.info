{'benchmarks_revision': ('cat: .gitid: No such file or directory',),
 'benchmarks_root': ('/homes/grads/dek61/SNIPER/benchmarks',),
 'cmdline': ['/homes/grads/dek61/SNIPER/sniper/run-sniper',
             '-n',
             '4',
             '-m',
             'localhost',
             '-d',
             '/scratch/nbp3/PARSEC_MULTIPROGRAM_SHELL/4cores/65/4/blackscholes_vips_streamcluster_swaptions',
             '-c',
             '/homes/grads/nbp3/SNIPER_CONFIGS/gainestown_65_4_4.cfg',
             '-s',
             'stop-by-icount:10000000000',
             '--curdir=/scratch/nbp3/PARSEC_MULTIPROGRAM_SHELL/4cores/65/4/blackscholes_vips_streamcluster_swaptions',
             '--trace-manual',
             '-g',
             '--traceinput/enabled=true',
             '-g',
             '--traceinput/emulate_syscalls=true',
             '-g',
             '--traceinput/num_apps=4',
             '-g',
             '--traceinput/trace_prefix=/tmp/tmpqqt_j1/run_benchmarks'],
 'git_revision': '68418dbe7cab15c65f1f4d300da0c505329beedd',
 'host': 'sb21',
 'pin_version': 'unknown',
 'rusage': (15791.555318000001,
            499.141119,
            198756,
            0,
            0,
            0,
            47319,
            0,
            0,
            8,
            1144,
            0,
            0,
            0,
            47721905,
            5296283),
 'snipercmd': ['bash',
               '-c',
               '/homes/grads/dek61/SNIPER/sniper/lib/sniper -c /homes/grads/dek61/SNIPER/sniper/config/base.cfg --general/total_cores=4 --general/output_dir=/scratch/nbp3/PARSEC_MULTIPROGRAM_SHELL/4cores/65/4/blackscholes_vips_streamcluster_swaptions --config=/homes/grads/dek61/SNIPER/sniper/config/nehalem65.cfg --config=/homes/grads/nbp3/SNIPER_CONFIGS/gainestown_65_4_4.cfg --traceinput/enabled=true --traceinput/emulate_syscalls=true --traceinput/num_apps=4 --traceinput/trace_prefix=/tmp/tmpqqt_j1/run_benchmarks -g --traceinput/stop_with_first_app=true -g --traceinput/restart_apps=false -g --hooks/numscripts=1 -g --hooks/script0name=/scratch/nbp3/PARSEC_MULTIPROGRAM_SHELL/4cores/65/4/blackscholes_vips_streamcluster_swaptions/sim.scripts.py -g --hooks/script0args='],
 't_elapsed': 6579.217423915863,
 't_start': 1463345317.38133,
 'tracecmds': [],
 'user': 'nbp3',
 'vmem': 1634041856L}
