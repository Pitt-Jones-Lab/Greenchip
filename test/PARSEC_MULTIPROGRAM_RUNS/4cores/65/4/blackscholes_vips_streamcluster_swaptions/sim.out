                                   | Core 0      | Core 1      | Core 2      | Core 3     
  Instructions                     |  2395635768 |  2510655553 |  2604928496 |  2487437521
  Cycles                           |  1315284082 |  1315284082 |  1315284082 |  1315284082
  IPC                              |        1.82 |        1.91 |        1.98 |        1.89
  Time (ns)                        |   494467700 |   494467700 |   494467700 |   494467700
  Idle time (ns)                   |       83379 |      575970 |       54317 |       82380
  Idle time (%)                    |        0.0% |        0.1% |        0.0% |        0.0%
Branch predictor stats             |             |             |             |            
  num correct                      |   242677112 |   241481509 |   186899333 |   240337778
  num incorrect                    |     6094469 |     4289999 |     2705645 |     4535378
  misprediction rate               |       2.45% |       1.75% |       1.43% |       1.85%
  mpki                             |        2.54 |        1.71 |        1.04 |        1.82
TLB Summary                        |             |             |             |            
  I-TLB                            |             |             |             |            
    num accesses                   |   324359680 |   333882493 |   374649348 |   330427370
    num misses                     |        8074 |        3411 |        2105 |        3167
    miss rate                      |       0.00% |       0.00% |       0.00% |       0.00%
    mpki                           |        0.00 |        0.00 |        0.00 |        0.00
  D-TLB                            |             |             |             |            
    num accesses                   |   787528510 |   818720983 |   979440652 |   815104856
    num misses                     |      133102 |      976646 |      145415 |      152589
    miss rate                      |       0.02% |       0.12% |       0.01% |       0.02%
    mpki                           |        0.06 |        0.39 |        0.06 |        0.06
  L2 TLB                           |             |             |             |            
    num accesses                   |      141176 |      980057 |      147520 |      155756
    num misses                     |       51962 |       58832 |       58870 |       53670
    miss rate                      |      36.81% |       6.00% |      39.91% |      34.46%
    mpki                           |        0.02 |        0.02 |        0.02 |        0.02
Cache Summary                      |             |             |             |            
  Cache L1-I                       |             |             |             |            
    num cache accesses             |   324359680 |   333882493 |   374649348 |   330427370
    num cache misses               |      231239 |      200944 |      124050 |      186161
    miss rate                      |       0.07% |       0.06% |       0.03% |       0.06%
    mpki                           |        0.10 |        0.08 |        0.05 |        0.07
  Cache L1-D                       |             |             |             |            
    num cache accesses             |   787528510 |   818720983 |   979440652 |   815104856
    num cache misses               |     8230594 |     9729471 |     6815366 |     9086872
    miss rate                      |       1.05% |       1.19% |       0.70% |       1.11%
    mpki                           |        3.44 |        3.88 |        2.62 |        3.65
  Cache L2                         |             |             |             |            
    num cache accesses             |     8923245 |    10399095 |     8401800 |     9721900
    num cache misses               |     1202948 |     1215709 |     2388938 |     1216847
    miss rate                      |      13.48% |      11.69% |      28.43% |      12.52%
    mpki                           |        0.50 |        0.48 |        0.92 |        0.49
DRAM summary                       |             |             |             |            
  num dram accesses                |    11674283 |           0 |           0 |           0
  average dram access latency (ns) |       69.66 |         inf |         inf |         inf
  average dram queueing delay      |       69.66 |        0.00 |        0.00 |        0.00
Coherency Traffic                  |             |             |             |            
  num loads from dram              |     1107641 |     1127550 |     2060520 |     1144800
  num loads from dram cache        |           0 |           0 |           0 |           0
  num loads from remote cache      |           0 |           0 |           0 |           0
